CHANGELOG.md

```diff
@@ -1,3 +1,15 @@
+## neptune-lightgbm 1.0.0
+
+ ### Changes
+ - `NeptuneCallback` now accepts a namespace `Handler` as an alternative to `Run` for the `run` argument. This means that
+   you can call it like `NeptuneCallback(run=run["some/namespace/"])` to log everything to the `some/namespace/`
+   location of the run.
+
+ ### Breaking changes
+ - We removed support for the legacy Neptune client API. Use it with `import neptune.legacy as neptune`.
+ - The integration uses now `Run.append()` method instead of `Run.log()` which is available since version 0.16.14
```

Comment:
Technically `append()` and `log()` don't operate on the `Run` object directly. Maybe we can just leave it out? Also, word order.

```suggestion
 - Instead of the `log()` method, the integration now uses `append()` which is available since version 0.16.14
```